modules/userId/index.js

```diff
@@ -755,7 +755,8 @@ function updateSubmodules() {
 
   // find submodule and the matching configuration, if found create and append a SubmoduleContainer
   submodules = addedSubmodules.map(i => {
-    const submoduleConfig = find(configs, j => j.name === i.name);
+    const submoduleConfig = find(configs, j => j.name && j.name.toLowerCase() === i.name.toLowerCase());
+    if (submoduleConfig && i.name !== submoduleConfig.name) submoduleConfig.name = i.name;
```

Comment:
should we throw a warning/info message here to let the publisher know they've used the wrong case? or since it's solved now it doesn't really matter?

Reply:
I can if you really want.  Let me know either way.

Reply:
I guess there's really no impact to using the wrong case now, so no real reason to spit out a log message. if i change my mind later, we can always add it then...